### PR TITLE
fix/force setProfile to use valid costCenter - [KI 945609]

### PR DESCRIPTION
**Pull request by giurigaud**
**What problem is this solving?**

Buyers cannot log in if they are associated with a deleted cost center

**How should this be manually tested?**

1- create Org
2- create CC1
3- create CC2
4- associate user 1 with CC1
5- associate user 1 with CC2
6- delete CC1
7- try to log in with user 1

**Screenshots or example usage:**

Before change
https://github.com/user-attachments/assets/c874c3a0-e458-4881-81d3-e6b52bda7a5e

After change

Uploading deleteCostCenter-afterChange.mov…

Screenshot 2024-10-02 at 11 48 42

Uploading deleteCostCenter-afterChange.mov…

### CODE REVIEW

File: node/service.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "stack": "nodejs",
   "memory": 256,
   "ttl": 60,
-  "timeout": 30,
+  "timeout": 45,
```
Comment (enzomerca) [inline node/service.json:5]: why are we changing the timeout? and why 45? also, the changelog says we are increasing it to 60.

**Review by enzomerca — COMMENTED** [review]
It looks like the fix still not working in the "after" video. Did you upload the correct video?

File: node/service.json (hunk 1 above)
Comment (giurigaud) [reply node/service.json:5]: I was getting a timeout error on this call, I don't know if this happens in prd too. I kept trying until I found a number that the timeout stopped. But if you have another suggestion, we can remove this one.

Screenshot 2024-09-09 at 14 43 22

**Comment by giurigaud** [comment]
> It looks like the fix still not working in the "after" video. Did you upload the correct video?

Uploading deleteCostCenter-afterChange.mov…

I'm sorry, this is the right video

**Review by enzomerca — APPROVED** [review]

**Pull request merged**